scripts/push_annotations_from_files.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""Push annotations from JSON files to Synapse entities"""
+
+import synapseclient
+import json
+
+def push_annotations_from_file(file_path, entity_id):
+    """Push annotations from a JSON file to a Synapse entity"""
+    
+    print(f"Pushing annotations from {file_path} to entity {entity_id}")
+    print("=" * 60)
+    
+    # Read the JSON file
+    try:
+        with open(file_path, 'r') as f:
+            annotations = json.load(f)
+        print(f"✅ Loaded annotations from {file_path}")
+        print(f"Annotations: {json.dumps(annotations, indent=2)}")
+    except Exception as e:
+        print(f"❌ Error reading file {file_path}: {e}")
+        return
+    
+    # Login to Synapse
+    syn = synapseclient.login()
+    
+    try:
+        # Get the entity
+        entity = syn.get(entity_id)
+        print(f"✅ Retrieved entity: {entity.name}")
+        
+        # Apply the annotations
+        # Convert single values to lists (Synapse expects lists)
+        synapse_annotations = {}
+        for key, value in annotations.items():
+            if isinstance(value, list):
+                synapse_annotations[key] = value
+            else:
+                synapse_annotations[key] = [value]
```

Comment:
Depending on what version of the client you are using, you may want to consider using the new OOP interface that we are moving towards as much of this stuff will be deprecated